server/routes/template.js

```diff
@@ -1,10 +1,28 @@
+import config from 'config'
+
 export default (content, helmet) => `
 <!doctype html>
 <html lang="en">
   <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js"></script>
+    <script>
+      WebFont.load({
+        google: {
+          families: ['Open+Sans:400italic,700,400:latin,latin-ext']
+        }
+      });
+    </script>
     ${helmet.meta.toString()}
     ${helmet.link.toString()}
-    <link rel="stylesheet" type="text/css" href="/bundle/main.css"/>
+    <script>
```

Comment:
In the other apps, we are supplementing the base google analytics with https://github.com/googleanalytics/autotrack 

I also added some additional configuration to track file downloads https://github.com/AusDTO/dto-digitalmarketplace-buyer-frontend/commit/92921ef8af31897d800f72e2d6dcea731e4b42af#diff-58a25b0ac70b2234f49baf128a42061dR10